Sources/Atomic.swift

```diff
@@ -107,80 +107,114 @@ internal struct UnsafeAtomicState<State: RawRepresentable> where State.RawValue
 /// `Lock` exposes `os_unfair_lock` on supported platforms, with pthread mutex as the
 // fallback.
 internal class Lock {
+	// Both `UnfairLock` and `PthreadLock` use `ManagedBufferPointer` to allocate inline
```

Comment:
It's the "abuse" of `ManagedBufferPointer` here that concerns me. I'd like to see us pursue the more standard approach first and only switch to a more complex, unintended approach when/if we have benchmarks that show that show a measurable difference.